pkg/processor/runtime/pypy/test/pypy_test.go

```diff
@@ -199,6 +199,13 @@ func (suite *TestSuite) TestOutputs() {
 	})
 }
 
+func (suite *TestSuite) TestStress() {
+
+	// Create Test stress with default stress-test configurations and require it to bee true
+	stressTestResult := suite.BlastHTTP(suite.GetDefaultStressRequest())
```

Comment:
Don't return booleans to indicate error, they hold no information about what went wrong. `BlastHTTP` should either return `error` or do the assertions itself and return nothing. I prefer the latter.